Protocol.md

```diff
@@ -627,6 +631,13 @@ subprotocol strings, and:
   SHALL validate that the negotiated subprotocol is present in the
   *subprotocols* field.
 
+Furthermore, the server SHALL validate that the *ping_interval* field
+contains a non-negative integer. If the value is `0`, the server SHALL
+NOT send WebSocket *ping* messages to the client. Otherwise, the server
```

Comment:
Yeah, I've added a SHOULD there only because there might be a good reason why someone doesn't want to massively ping clients all the time. But it's enabled in the Python server and there's not even an option to turn it off so far.